functions/imagemagick/src/main/java/com/example/functions/imagemagick/ImageMagick.java

```diff
@@ -86,7 +92,7 @@ public void accept(GcsEvent gcsEvent, Context context) {
           LOGGER.info(String.format("Detected %s as OK.", gcsEvent.getName()));
         }
       }
-    } catch (Exception e) {
+    } catch (IOException e) {
       LOGGER.info(String.format("Error with Vision API: %s", e.getMessage()));
     }
```

Comment:
Can you be more specific?
Is there a form of mitigation?
Should you even be catching this?  (rather than providing a warning it might happen and let it propagate)

Reply:
Switched to a `LOGGER.log()` format that allows printing the entire exception.

("catching this and letting it propagate" is probably the _right_ answer, but **will not compile** without a cast to `RuntimeException` due to [this issue](https://github.com/GoogleCloudPlatform/functions-framework-java/issues/21) until the release is updated)